src/NuGet.Core/NuGet.Packaging.Extraction/NuGet.Packaging.Extraction.csproj

```diff
@@ -307,6 +320,20 @@
     </EmbeddedResource>
   </ItemGroup>
 
+  <Target Name="ILMergeNPE" AfterTargets="Build" Condition="'$(BuildingInsideVisualStudio)' != 'true' and '$(SkipILMergeOfNPE)' != 'true'" >
+    <ItemGroup>
+      <BuildArtifacts Include="$(OutputPath)\*.dll" Exclude="@(MergeExclude)" />
+    </ItemGroup>
+    <PropertyGroup>
+      <PathToBuiltNPE>$(OutputPath)NuGet.Packaging.Extraction.dll</PathToBuiltNPE>
+      <IlmergeCommand>$(ILMergeExePath) /lib:$(OutputPath) /t:library /out:$(ArtifactsDirectory)$(VsixOutputDirName)\NuGet.Packaging.Extraction.dll @(MergeAllowDup -> '/allowdup:%(Identity)', ' ')</IlmergeCommand>
```

Comment:
setting the output to artifacts\VS15 means we'll archive the dll and pdb on ddfiles, plus save it to the build artifacts files.  Can you output to `$(OutputPath)\ilmerge\` instead?

Reply:
I agree here. Don't think we need it in artifacts. 

Is it better if we compile into one assembly instead of ilmerging?
Is that harder?

Reply:
FWIW, NuGet.Build.Tasks.Pack also ilmerges it assemblies, and does so in the bin\ilmerge folder, not the VS15 folder. THough with a 30 second look, I can't figure out how: https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.csproj

Reply:
To the rescue!

https://github.com/NuGet/NuGet.Client/blob/01fbfa44d1c93b036132bcb60ca83d0489f165d7/src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.csproj#L88-L113

Reply:
outputting to bin\ilmerge\ now...
outputdirectory was not set "after" build...so having it under there was problematic.

yes, i had grabbed a bunch of targets content from the pack tasks csproj to fix signing, etc...